datastore/src/main/java/org/jboss/pnc/datastore/repositories/BuildConfigurationAuditedRepositoryImpl.java

```diff
@@ -203,6 +205,8 @@ public BuildConfigurationAudited queryById(IdRev idRev) {
 
     @Override
     public List<IdRev> searchIdRevForBuildConfigurationName(String buildConfigurationName) {
+        buildConfigurationName = buildConfigurationName.replaceAll("\\*", "%");
```

Comment:
shouldn't this be in the provider? Here I think it's in too deep level.

Reply:
I think it could be in both. I prefered it here as * is more common than % and wanted to hide the logic. It is described clearly in the javadoc